### DISCUSSION TITLE
Showing error without reason

### DISCUSSION THREAD

**Discussion by adnanattar** [community/community#21191] [GitHub Education]
Hello, I'm trying to submit my ID proof by taking picture with my mobile back camera then also while submitting it showing me error as 

Your application cannot be reviewed until you fix one of the following:
Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

Please help to solve this error

**Comment by c9long** [comment]
Ran into the same issue and a multitude of others. Github Education support likely will not help you. You've just got to keep trying by taking multiple different screenshots and uploading all of them, or take photos of other types of ID. 

Their staff write only the same messages as the bot that scans your ID in the first place:
![image](https://user-images.githubusercontent.com/109810526/181136079-0ed216e6-bc8b-4e17-86b7-dc2852568702.png)
